pkg/containerd/scripts/pkg-rpm-build.sh

```diff
@@ -66,6 +66,12 @@ if [ -n "$(xx-info variant)" ]; then
   pkgoutput="${pkgoutput}/$(xx-info variant)"
 fi
 
+case "$PKG_RELEASE" in
+  centos7|centos8|centos9|oraclelinux*)
```

Comment:
Do we need `rhel*` here as well?

I guess `centos*` (any version) would work for this going forward (we kept btrfs for 1.6 for now in containerd-packaging, to not drop it in a patch release)

Reply:
Ah right `centos*` would be enough. We don't publish specific `rhel` packages atm, only the following: https://github.com/docker/packaging/blob/55b6e3984539bf162dabdb697421b062c6840693/common/packages.mk#L19

Reply:
Yeah, we don't build for RHEL ourselves, but I know IBM builds RHEL packages (for s390x and/or ppc64le), so was wondering if we need to account for that

Reply:
I think it's fine for now, at some point when I get to porting the Mirantis build matrix over explicit RHEL might make sense, but it seems easy enough to grep for until then.